nix/tools/mercurial.nix

```diff
@@ -45,12 +45,17 @@ in python2Packages.buildPythonApplication {
   postInstall = ''
     mkdir -p $out/etc/mercurial
     cat > $out/etc/mercurial/hgrc <<EOF
+    [ui]
+    username = Richard Hendricks <rhendricks@pied-piper.com>
```

Comment:
Nice name!

Is it needed for the mock repo?

Reply:
yes to have an author for the commit

Reply:
is "username = Moz Illa <milla@mozilla.org>" ok for you guys ?